tests/manual-test-cases/Group5-Functional-Tests/5-28-VICAdmin-Isolated.robot

```diff
@@ -0,0 +1,88 @@
+# Copyright 2016-2018 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+*** Settings ***
+Documentation  Test 5-28 - VICAdmin Isolated
+Resource  ../../resources/Util.robot
+Suite Setup  Deploy Testbed With Static IP
+Suite Teardown  Teardown VCH With No WAN
+Default Tags
+
+*** Keywords ***
+Deploy Testbed With Static IP
+    Setup VCH With No WAN
+    Deploy VCH With No WAN
+
+Setup VCH With No WAN
```

Comment:
I see this is setting up VC and not VCH, so would rename it to `Setup VC With No WAN`

Reply:
Done.